content/en/docs/languages/python/basics.md

```diff
@@ -153,13 +153,13 @@ example directory, running this command regenerates the appropriate file rather
 than creates a new one. The generated code files are called
 `route_guide_pb2.py` and `route_guide_pb2_grpc.py` and contain:
 
-- classes for the messages defined in route_guide.proto
-- classes for the service defined in route_guide.proto
-   - `RouteGuideStub`, which can be used by clients to invoke RouteGuide RPCs
-   - `RouteGuideServicer`, which defines the interface for implementations
+- classes for the messages defined in route_guide_pb2.py
```

Comment:
The _messages_ are defined in `route_guide.proto` not `route_guide_pb2.py` -- the latter contains the generated code, not the original message definition.

Similar remarks apply to the other changes you've made.